includes/class-wc-privacy-exporters.php

```diff
@@ -348,4 +348,49 @@ protected static function get_download_personal_data( $download ) {
 
 		return $personal_data;
 	}
+
+	/**
+	 * Finds and exports customer tokens by email address.
+	 *
+	 * @since 3.4.0
+	 * @param string $email_address The user email address.
+	 * @param int    $page  Page.
+	 * @return array An array of personal data in name value pairs
+	 */
+	public static function customer_tokens_exporter( $email_address, $page ) {
+		$user           = get_user_by( 'email', $email_address ); // Check if user has an ID in the DB to load stored personal data.
+		$data_to_export = array();
+
+		if ( ! $user instanceof WP_User ) {
+			return array(
+				'data' => $data_to_export,
+				'done' => true,
+			);
+		}
+
+		$tokens = WC_Payment_Tokens::get_tokens( array(
+			'user_id' => $user->ID,
+			'limit'   => 10,
+			'page'    => $page,
+		) );
+
+		if ( 0 < count( $tokens ) ) {
+			foreach ( $tokens as $token ) {
+				$data_to_export[] = array(
+					'group_id'    => 'woocommerce_tokens',
```

Comment:
Not sure if we want to group this separately or in `woocommerce_customer`.

Reply:
Separate is fine.